### PR TITLE
introduce plain_to_s

### PR DESCRIPTION
**Pull request by privat**
In order to have a more POLA `to_s` on collections, it is required that a new method with the old behavior is provided for clients that need it (especially the interpreter that use it for superstrings).

the compiler use `native_to_s` for superstrings but `c_src/nitg` still use `to_s` on array, so a regeneration of the bootstrap is required before changing the behavior of `to_s` on collections.

### CODE REVIEW

**Comment by xymus** [comment]
What will `Collection::to_s` do? insert a space, a comma?

+1

**Comment by privat** [comment]
I plan to have `"[{join(",")}]"` as a general Collection::to_s, so `assert [1,2,3].to_s == "[1,2,3]"`

**Comment by ablondin** [comment]
Thank you for this feature, it will simplify debugging!

**Comment by BlackMinou** [comment]
+1, agree with @ablondin

File: lib/standard/string.nit [hunk 1]
```diff
@@ -2225,6 +2225,12 @@ redef class Collection[E]
 		return s.to_s
 	end
 
+	# Concatenate element without separators
+	fun plain_to_s: String
```
Comment (Morriar) [inline lib/standard/string.nit]: I don't like the name. But I have nothing better to suggest.

**Comment by Morriar** [comment]
+1

**Comment by jpages** [comment]
Good feature, +1

**Comment by privat** [comment]
update: try to pass last test

**Pull request merged**